js/history.js

```diff
@@ -3,17 +3,80 @@ Fliplet.Registry.set('history-stack', (function() {
   var stack = [];
   var currentIndex = 0;
 
+  const columnsInfo = {
+    original: () => stack[0].data[0],
+    reference: [],
+    getReference: () => {
+      if (!columnsInfo.reference.length) {
+        columnsInfo.reference = columnsInfo.original();
+      }
+      return columnsInfo.reference;
+    },
+    current: [],
```

Comment:
`current` is only used internally within this class and not used publicly. So, I'd make it an private variable instead. e.g.

```
const current = [];

function moveColumns() {
  // This uses the current array
}

return {
  moveColumns
};
```